lib/extensions/string_iec.rb

```diff
@@ -0,0 +1,12 @@
+class String
+  IEC_SIZE_SUFFIXES = %w(Ki Mi Gi Ti).freeze
+  # https://github.com/jason-riddle/deploying-with-kubernetes/wiki/Conversion-Units
+  def parse_iec_number
```

Comment:
If you are going the monkey patching route, then I wouldn't use the word parse anymore... perhaps `to_iec_integer`